package.json

```
@@ -1,11 +1,11 @@
 {
-  "name": "notion-exporter",
-  "author": "Yann Bolliger",
+  "name": "@onebeyond/notion-exporter",
+  "author": "Yann Bolliger/One Beyond",
```

Author shoulbe a single person and contributors can be multiple people: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#people-fields-author-contributors

```suggestion
  "author": "Yann Bolliger",
  "contributors: ["One Beyond"],
```